### PR TITLE
Set the rollupReplace default param - removes warning message.

### PR DESCRIPTION
**Pull request by blckmn**
- will need to confirm "true" has no impact, otherwise should set it to false.

### CODE REVIEW

**Comment by blckmn** [comment]
AUTOMERGE: (FAIL)
 - github identifies PR as mergeable -> FAIL
 - assigned to a milestone -> FAIL
 - cooling off period lapsed -> FAIL
 - commit count less or equal to three -> PASS
 - `Don't merge` label NOT found -> PASS
 - at least one `RN:` label found -> FAIL
 - `Tested` label found -> FAIL
 - assigned to an approver -> PASS
 - approver count at least three -> FAIL

**Review by haslinghuis — APPROVED** [review]

**Comment by nerdCopter** [comment]
How to test? (true vs false)

**Comment by haslinghuis** [comment]
Fixes:

• service worker @rollup/plugin-replace: 'preventAssignment' currently defaults to false. It is recommended to set this option totrue, as the next major version will default this option to true.

See `https://github.com/sveltejs/sapper-template/issues/302`

**Review by nerdCopter — APPROVED** [review]

**Pull request merged**
